### ISSUE TITLE
Standalone Linux Cluster on Azure

### ISSUE THREAD

**Issue by Pothulapati** [Azure/service-fabric-issues#1204]
This [Document](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-deploy-anywhere#create-service-fabric-standalone-clusters-on-premises-or-with-any-cloud-provider) says that you can't create a standalone Linux Cluster on Azure or other cloud provider. Just like there are scripts for windows, Aren't there scripts through which a linux SF cluster can be made without using the Azure Service.

I'm really shocked to see that you can't create a standalone SF linux cluster(as the doc says). Please prove me wrong.

Thanks :smile: 

Thanks.

**Comment by suhuruli** [comment]
We do not have the scripts like we do for Windows currently.

**Comment by Pothulapati** [comment]
Yeah, But It's a pretty normal use-case to create standalone Linux clusters. An Azure doc with installation scripts and quick instructions would definitely be enough and do the work. :smiley:

**Comment by suhuruli** [comment]
@dkkapur

**Comment by MZDN** [comment]
@Pothulapati 
These links might help:
https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started-linux
https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-tutorial-create-vnet-and-linux-cluster

**Comment by chenzuo** [comment]
Is there a plan to support standalone clusters on Linux?

**Comment by masnider** [comment]
@dkkapur to comment on plans, timeframes, and alternatives.
